### PR TITLE
Revised from previous PR90: Add contains function to RowIndex

### PR DESCRIPTION
**Pull request by kodeschreiber**
- Added a "contains" function to the RowIndex trait
- Defined the contains funtion for &str and usize columns

### CODE REVIEW

File: src/cursor.rs [hunk 1]
```diff
@@ -30,6 +30,9 @@ pub struct Row {
 
 /// A type suitable for indexing columns in a row.
 pub trait RowIndex: std::fmt::Debug {
+    /// Check to see if the row contains a Column
```
Comment (IvanUkhov) [inline src/cursor.rs:33]: “Check if the row contains a column.”

**Review by IvanUkhov — COMMENTED** [review]
I think, to make it useful, you have to define `Row::contains`.

**Comment by IvanUkhov** [comment]
Thank you! I usually add people to the authors list. Do you have any name and optionally email that you would like to see there?

**Comment by kodeschreiber** [comment]
I'm good with my username being put in the list. Otherwise, I'm fine with just contributing :)

**Pull request merged**